.github/workflows/build.yml

```diff
@@ -0,0 +1,94 @@
+name: build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  unit-tests:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+
+    name: Unit Tests
+
+    runs-on: ${{ matrix.os }}
+
+    outputs:
+      job-status: ${{ job.status }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: lerna install
+        run: |
+          npm install lerna@^3.22.0
+          ./node_modules/.bin/lerna bootstrap 2>&1
```

Comment:
Actually we will need to change this one to `npm install`.

Reply:
Changed in `publish.yml` as well.

Reply:
👍 

The reason is because we do something a little tricky in the markdown transform: before calling `lerna bootstrap` we need to download the latest versions of CiceroMark/TemplateMark coo models (you can see in the `package.json` what happens on install to get an idea.